src/main/java/com/guciowons/shoppingbasket/Product/Product.java

```diff
@@ -0,0 +1,45 @@
+package com.guciowons.shoppingbasket.Product;
+
+import java.math.BigDecimal;
+
+public class Product {
+    private final int id;
+    private String name;
+    private String description;
+    private BigDecimal cost;
```

Comment:
To jest bardziej `price`, a nie `cost`?

https://www.investopedia.com/ask/answers/101314/what-difference-between-cost-and-price.asp